rhosp_container/licenses/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This is already contained in this repository (at the project root), would be better to retrieve it from there (or even better, extract it out of the source tar). What if we decide to change the `swift-scality-backend` license, and forget to change this file?

Reply:
Right, I could not find the way of doing this from the Dockerfile till now :)  Fixed now

Reply:
This is done by specifying the Dockerfile from the parent directory to the "docker build" command, with the "-f" option